wp-content/themes/engage/src/Managers/Theme.php

```diff
@@ -124,8 +133,11 @@ public function addToContext( $context ) {
 		$context['secondaryMenu'] = new \Timber\Menu('secondary-menu');
 		$context['quickLinks'] = new \Timber\Menu('quick-links');
 		$context['site'] = new \Timber\Site();
-        $context['footerMenu'] = new \Timber\Menu('footer-menu');
-        $context['footerWidgets'] = \Timber::get_widgets('sidebar-footer');
+    $context['footerMenu'] = new \Timber\Menu('footer-menu');
+    $context['footerWidgets'] = \Timber::get_widgets('sidebar-footer');
+		if (is_singular('page') || is_singular($post_types)) {
```

Comment:
I don't see where $post_types is defined. Were you wanting to define an array of allowed post types that the newsletter would be attached to?